### DISCUSSION TITLE
Native Apps

### DISCUSSION THREAD

**Discussion by thomastthai** [teamhanko/hanko#500] [General]
How would native apps make use of hanko? The examples are all web-based frontends.

**Comment by FlxMgdnz** [comment]
Hi, native apps can use Hanko backend or Hanko Cloud for authentication, incl. passkeys. We did a PoC for Android and are working on the same for iOS:

- https://docs.hanko.io/guides/mobile_guide
- https://github.com/teamhanko/hanko-android-example

**Reply by thomastthai** [reply]
@FlxMgdnz, thanks for the quick reply! Is there an SDK for Flutter?

**Reply by FlxMgdnz** [reply]
Not yet unfortunately. But an SDK is not required. Integration can be done without an SDK and should be straightforward when following the mobile guide.

We'll do Flutter and React Native examples as soon as possible, but it may take a few more weeks.

**Reply by thomastthai** [reply]
@FlxMgdnz, looking forward to the SDK!

**Comment by jonasbadstuebner** [comment]
Hey there, any updates on the Flutter SDK/example?
I'd love to test Hanko with a Flutter App.

**Comment by rohiniaru** [comment]
Hi, is there any example on this on how to integrate with React Native expo app?

**Comment by vdevc** [comment]
The mobile guide link previously linked at: https://docs.hanko.io/guides/mobile_guide does not exists.
I have found this link while trying to validate my library developed in Flutter.

**Reply by FlxMgdnz** [reply]
Thanks, yes it was outdated and we decided to take it offline for now.
